service/state/interface.go

```diff
@@ -28,4 +30,7 @@ type Accessor interface {
 	// Celestia network and blocks until the tx is included in
 	// a block.
 	SubmitTx(ctx context.Context, tx Tx) (*TxResponse, error)
+
+	// Transfer sends the given amount of coins from default wallet of the node to the given account address.
+	Transfer(ctx context.Context, to types.AccAddress, amount types.Int, gasLimit uint64) (*TxResponse, error)
```

Comment:
Btw, you defined an alias, but you are not using it here.

Reply:
Anyway, I suggest to use existing Address here and assert the correct type is passed